modules/installation-special-config-encrypt-disk-tang.adoc

```diff
@@ -4,92 +4,96 @@
++
 [source,terminal]
 ----
-$ sudo yum install clevis -y
+$ sudo yum install clevis
 ----
 
 [start=5]
-. Generate a thumbprint from the Tang server.
-
-.. In the following command, replace the value of `url` with the Tang server URL:
+. On the {op-system-base} 8 machine, run the following command to generate a thumbprint of the exchange key. Replace `\http://tang.example.com:7500` with the URL of your Tang server:
 +
 [source,terminal]
 ----
-$ echo nifty random wordwords \
-     | clevis-encrypt-tang \
-       '{"url":"https://tang.example.org"}'
+$ clevis-encrypt-tang '{"url":"http://tang.example.com:7500"}' < /dev/null > /dev/null <1>
 ----
+<1> In this example, `tangd.socket` is listening on port `7500` on the Tang server.
++
+[NOTE]
+====
+The `clevis-encrypt-tang` command is used in this step only to generate a thumbprint of the exchange key. No data is being passed to the command for encryption at this point, so `/dev/null` is provided as an input instead of plain text. The encrypted output is also sent to `/dev/null`, because it is not required for this procedure.
+====
 +
 .Example output
 [source,terminal]
 ----
 The advertisement contains the following signing keys:
 
-PLjNyRdGw03zlRoGjQYMahSZGu9
+PLjNyRdGw03zlRoGjQYMahSZGu9 <1>
 ----
-
-.. When the `Do you wish to trust these keys? [ynYN]` prompt displays, type `Y`, and the thumbprint is displayed:
+<1> The thumbprint of the exchange key.
 +
-.Example output
-[source,terminal]
-----
-eyJhbmc3SlRyMXpPenc3ajhEQ01tZVJiTi1oM...
-----
+When the `Do you wish to trust these keys? [ynYN]` prompt displays, type `Y`.
++
+[NOTE]
+====
+{op-system-base} 8 provides Clevis version 15, which uses the SHA-1 hash algorithm to generate thumbprints. Some other distributions provide Clevis version 17 or later, which use the SHA-256 hash algorithm for thumbprints. You must use a Clevis version that uses SHA-1 to create the thumbprint, to prevent Clevis binding issues when you install {op-system-first} on your {product-title} cluster nodes.
+====
 
 . Create a Base64 encoded file, replacing the URL of the Tang server (`url`) and thumbprint (`thp`) you just generated:
 +
 [source,terminal]
 ----
 $ (cat <<EOM
 {
- "url": "https://tang.example.com",
- "thp": "PLjNyRdGw03zlRoGjQYMahSZGu9"
+ "url": "http://tang.example.com:7500", <1>
+ "thp": "PLjNyRdGw03zlRoGjQYMahSZGu9" <2>
 }
 EOM
 ) | base64 -w0
 ----
+<1> Specify the URL of a Tang server. In this example, `tangd.socket` is listening on port `7500` on the Tang server.
+<2> Specify the exchange key thumbprint, which was generated in a preceding step.
 +
 .Example output
 [source,terminal]
 ----
 ewogInVybCI6ICJodHRwczovL3RhbmcuZXhhbXBsZS5jb20iLAogInRocCI6ICJaUk1leTFjR3cwN3psVExHYlhuUWFoUzBHdTAiCn0K
 ----
 
-. In the `openshift` directory, create master or worker files to encrypt disks for those nodes.
+. Create machine config files to encrypt the boot disks for the control plane or compute nodes using the Tang encryption mode. 
```

Comment:
It is better to prompt user to generate k8s manifests file before injecting the machineconfig files.

Reply:
Thank you for highlighting this. Resolved.